sequence_processing_pipeline/Pipeline.py

```diff
@@ -19,6 +19,20 @@
 
 
 class Pipeline:
+    sif_header = ['sample_name', 'collection_timestamp', 'elevation', 'empo_1',
```

Comment:
Would it be better to make these two a dict? In the future, it might be simpler to update if necessary.

Reply:
I thought about that, but in this case I thought it might be important to preserve column order, since that issue recently came up with another file. However, I'm happy to switch to dict if that isn't likely to be the case here.

Reply:
ohh I see what you are saying.